wagtail/admin/views/generic/base.py

```diff
@@ -41,9 +41,17 @@ class WagtailAdminTemplateMixin(TemplateResponseMixin, ContextMixin):
     page_title = ""
     page_subtitle = ""
     header_icon = ""
-    # Breadcrumbs are opt-in until we have a design that can be consistently applied
-    _show_breadcrumbs = False
+
     breadcrumbs_items = [{"url": reverse_lazy("wagtailadmin_home"), "label": _("Home")}]
+    """
+    The base set of breadcrumbs items to be displayed on the page.
+    The property can be overridden by subclasses and viewsets to provide
+    custom base items, e.g. page tree breadcrumbs or add the "Snippets" item.
+
+    Views should copy and append to this list in :meth:`get_breadcrumbs_items()`
+    to define the path to the current view.
+    """
```

Comment:
We probably should name this `base_breadcrumbs_items` instead. I initially only followed the existing patterns for `page_title`, `page_subtitle`, and `header_icon` having their respective `get_foo` methods.

I wasn't sure how the API would eventually take shape once we migrated all the views and made the page views use the same mechanism. I think it's starting to stabilise – but the role of the `breadcrumbs_items` list is more like the base items that can be easily overridden from a higher abstraction e.g. `SnippetViewSet` to provide the "Snippets" menu item, while the actual view defines the path to itself in the `get_breadcrumbs_items()` method.